pip-requirements-doc

```diff
@@ -3,4 +3,7 @@ matplotlib
 scipy
 pillow
 jplephem
+sphinx<1.7
```

Comment:
What happens with Sphinx 1.7?

Reply:
As I recall that's in reality 1.7.1. https://github.com/sphinx-doc/sphinx/issues/4689

It hasn't yet been followed up whether the new release is working for us (I recall seeing some other failures with it?). Anyway, a comment for it would be nice, so we know when to remove the limitation.